bin/dartdoc.dart

```diff
@@ -69,7 +69,7 @@ main(List<String> arguments) async {
   logInfo("Generating documentation for '${config.topLevelPackageMeta}' into "
       "${outputDir.absolute.path}${Platform.pathSeparator}");
 
-  Dartdoc dartdoc = await Dartdoc.withDefaultGenerators(config, outputDir);
+  Dartdoc dartdoc = await Dartdoc.withDefaultGenerators(config);
```

Comment:
Style question: since you're creatine an instance here, I'd have expected a named factory constructor instead of a static; any reason for that?

EDIT: like `withoutGenerators(..)`

Reply:
Ah, yes, right!  I didn't notice the `await`.  Thanks for clarifying. 👍